### PR TITLE
Housekeeping: Upgrade deprecated configs

### PR DESCRIPTION
**Pull request by KCeh**
## Summary

Small update to keep project up to date.

**Related issue**: 

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [x] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

File: dbinspector-no-op/publish.gradle [hunk 1]
```diff
@@ -6,9 +6,9 @@ task sourcesJar(type: Jar) {
     from android.sourceSets.main.kotlin.srcDirs
 }
 
-task javadocsJar(type: Jar, dependsOn: "dokkaJavadoc") {
-    archiveClassifier.set("javadoc")
-    from dokkaJavadoc.outputDirectory
+task javadocsJar(type: Jar, dependsOn: "dokkaGenerate") {
+    getArchiveClassifier().set("javadoc")
+    from layout.buildDirectory.dir("javadoc")
```
[the remark this answers is not in the document]
Comment (KCeh) [reply dbinspector-no-op/publish.gradle:11]: from dokka 2.0+ there is no taks dokkaJavadoc
